debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -959,6 +959,16 @@ public static AutoCreateMode parse(String value, String defaultValue) {
             .withDefault(Boolean.TRUE)
             .withValidation(Field::isBoolean, PostgresConnectorConfig::validateFlushLsnSource);
 
+    public static final Field READ_ONLY_CONNECTION = Field.create("read.only")
```

Comment:
Given the option name we should as a follw-up task verify that it really works on read-only database and if not then make the necessary changes.

Reply:
>  Postgres connector runs two queries which are not supported in recovery mode (i.e. on read replicas).

Is [DBZ-7181](https://issues.redhat.com/browse/DBZ-7181)  absolves to the doubt?

Reply:
Partially. We need to add a guard condition on slot/publication creation. It should not be attempted to be created on `read.only` connection, but it should be a separate Jira issue.